src/app/anuncio/[id]/page.tsx

```
@@ -1,67 +1,73 @@
+  useEffect(() => {
+    if (!id) return;
+    (async () => {
+      try {
+        const { data, error } = await supabase
+          .from('announcements')
+          .select('*, user:profiles(email)')
+          .eq('id', id)
+          .single();
+        if (error) throw new Error(error.message);
+        setAnnouncement(data);
+      } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)